modules/MIPS/generate_MIPS_xml.php

```diff
@@ -1,69 +1,57 @@
+<html>
+<body style="background-color:#fedf98;">
```

Comment:
Ehmmmm there is a general way going this

Reply:
See here with the class property `body_top` and referenced style file to pickup the default theme https://github.com/LibreHealthIO/lh-ehr/blob/master/interface/main/messages/messages.php

Reply:
@muarachmann The modal box looks like this after I added class and style file . : /
May I create a different class for it in the stylesheet ?

![Screenshot (342)](https://user-images.githubusercontent.com/33698440/54261566-187e8f80-4592-11e9-8142-219c3f1cfe81.png)

Reply:
Weird.. what theme are you on @Priyanka488 ?

Reply:
@muarachmann I just added base.css for styling.

![Capture](https://user-images.githubusercontent.com/33698440/54261988-2254c280-4593-11e9-9b9a-cdadf731ee83.PNG)

![Capture1](https://user-images.githubusercontent.com/33698440/54261994-2680e000-4593-11e9-8e58-052fa3631d39.PNG)